### ISSUE TITLE
2019-11-20 tc017 M4SD if FLGACCEPTKELCRIT is specified with a parameter not support by model computation engine crashes

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#31]
Test case tc017 M4SD was executed with mct$FLGACCEPTKELCRIT set to 
"KELRSQ>=0.9,KELNOPT>=3,AUCXPCTP<=20.0"

However, AUCXPCTP is not computed by the M4SD model.

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in `[.data.frame`(computation_df, , flag_df$VAR[f]) : 
  undefined columns selected
In addition: Warning messages:
1: In run_M4_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
2: In run_M4_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in `[.data.frame`(computation_df, , flag_df$VAR[f]) : 
  undefined columns selected 
```
When FLGACCEPTKELCRIT is reset to "KELRSQ>=0.9,KELNOPT>=3" and run_computation re-executed, the computation runs as expected.

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This error no longer occurs as of 02b2470.
